src/components/TableCell.re

```diff
@@ -0,0 +1,16 @@
+module Style = {
+  open Css;
+
+  let td =
+    style([
+      padding2(~v=rem(0.5), ~h=rem(0.0)),
+      overflow(hidden),
+      textOverflow(ellipsis),
+      fontSize(rem(0.95)),
+      color(hex(Theme.Colors.text)),
+      selector(":not(:last-child)", [paddingRight(rem(1.5))]),
+    ]);
+};
+
+[@react.component]
+let make = (~children) => <td className=Style.td> children </td>;
```

Comment:
aah nice, I think we could move this to AstroDesign. As it's likely to have tables in other projects

Reply:
als;dfas nvm my craziness

Reply:
kkkkkkkkkkkkkk